docs/tutorials/onnx2tensorrt.md

```diff
@@ -25,45 +25,41 @@
 
 ```bash
 python tools/deployment/onnx2tensorrt.py \
+    ${CONFIG} \
     ${MODEL} \
     --trt-file ${TRT_FILE} \
     --input-img ${INPUT_IMAGE_PATH} \
-    --shape ${IMAGE_SHAPE} \
+    --shape ${INPUT_IMAGE_SHAPE} \
+    --min-shape ${MIN_IMAGE_SHAPE} \
     --max-shape ${MAX_IMAGE_SHAPE} \
-    --mean ${IMAGE_MEAN} \
-    --std ${IMAGE_STD} \
-    --dataset ${DATASET_NAME} \
     --workspace-size {WORKSPACE_SIZE} \
     --show \
     --verify \
 ```
 
 Description of all arguments:
 
+- `config` : The path of a model config file.
 - `model` : The path of an ONNX model file.
 - `--trt-file`: The Path of output TensorRT engine file. If not specified, it will be set to `tmp.trt`.
 - `--input-img` : The path of an input image for tracing and conversion. By default, it will be set to `demo/demo.jpg`.
 - `--shape`: The height and width of model input. If not specified, it will be set to `400 600`.
+- `--min-shape`: The minimum height and width of model input. If not specified, it will be set to the same as `--shape`.
```

Comment:
I suggest to also clean the `min-shape`, `shape`, and `max-shape`, which can also use the default value in config.

Reply:
min-shape and max-shape for converting from ONNX to TensorRT which can not be removed.